exe-unit/src/service/metrics.rs

```diff
@@ -95,7 +95,7 @@ impl Handler<GetMetrics> for MetricsService {
         let mut metrics = Vec::with_capacity(self.usage_vector.len());
 
         for name in self.usage_vector.iter() {
-            let metric = self
+            /*let metric = self
```

Comment:
How about making this feature enabled, e.g. with `#[cfg(feature = "sgx")]`, `#[cfg(not(feature = "sgx"))]`

Reply:
Oops, this commit was included here by mistake.

Reply:
Rebased without this commit, it'll be in a separate PR.